src/OpenTelemetry.Instrumentation.Process/OpenTelemetry.Instrumentation.Process.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net462;netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
nit - unless we know for sure we need all these targets, we can start with netstandard2.0.

Reply:
removed others